### DISCUSSION TITLE
部署到 Vercel 时最后一步显示 “环境变量超过了 4 kb 限制” 该如何解决？

### DISCUSSION THREAD

**Discussion by iamalexblue** [spencerwooo/onedrive-vercel-index#122] [Q&A] [answered]
# 部署到 Vercel 时最后一步显示 “环境变量超过了 4 kb 限制” 该如何解决？
部署到 Vercel 时报错如下，在对应网站说明中并未找到如何减小该文件大小，请问该如何解决呢？

`
Error: The Environment Variables provided for your Serverless Functions exceed the limit of 4KB. Reduce their size to continue. Learn More: https://vercel.link/env-vars-limit
`

![Error ScreenShot](https://user-images.githubusercontent.com/41639269/133982212-530dc346-abf5-4442-ade2-ed4f62ec13e7.png)

**Comment by iamalexblue** [comment]
叨扰大佬了，小白一枚折腾忘了看 issue，抱歉抱歉

**Comment by spencerwooo** [comment, marked as answer]
新版本不会遇到此类问题了
